src/Settings/Sidebar.jsx

```diff
@@ -0,0 +1,94 @@
+let { onClick, activeTab } = props;
+
+activeTab = activeTab ?? "identity";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border-right: 1px solid #eceef0;
+  gap: 24px;
+  width: 264px;
+  height: 100%;
+`;
+
+const Title = styled.h2`
+  font: var(text-l);
+  font-weight: 600;
+  padding: 64px 24px 24px;
+`;
+
+const MenuItem = styled.div`
```

Comment:
We switched these to buttons in the moderation sidebar for better accessibility.

Reply:
@gabehamilton if you don't mind I'd rather do this improvement in the next PR